package-lock.json

```diff
@@ -1,12 +1,12 @@
 {
   "name": "linkding",
-  "version": "1.11.1",
```

Comment:
Let's revert changes in this file. It should probably be updated, but not in this PR.

Reply:
Reverted change